docs/config.rst

```diff
@@ -603,17 +609,20 @@ Rendering
 
         renders['day'] = {
             'world': 'exmaple',
+            'dimension': 'overworld',
```

Comment:
This isn't needed, that's the default.

Reply:
Removed in f8eba0f.

All trough I though my renders had issues when I left that out, but that's probably an issue on my part then. Might look into that again later.